### PR TITLE
Expand screen width

### PR DESCRIPTION
**Pull request by imbyungjun**
- increase width to 1200px
- increase height

### CODE REVIEW

**Comment by donggyu04** [comment]
deploy this

**Comment by donggyu04** [comment]
deploy this

**Comment by junoyoon** [comment]
build this

**Comment by songeunwoo** [comment]
LGTM^^

File: ngrinder-controller/src/main/java/org/ngrinder/home/controller/HomeApiController.java [hunk 1]
```diff
@@ -108,6 +108,11 @@ public void init() {
 		return Collections.emptyList();
 	}
 
+	@GetMapping("/messagesources/{locale}")
+	public Map<String, String> getMessageSources(@PathVariable String locale) {
+		return homeService.getUserMessageSources(locale);
```
Comment (junoyoon) [inline ngrinder-controller/src/main/java/org/ngrinder/home/controller/HomeApiController.java]: Rename it ```getUserDefinedMessageSources```

File: ngrinder-controller/src/main/java/org/ngrinder/common/constant/CacheConstants.java [hunk 2]
```diff
@@ -26,6 +26,7 @@
 
 	String CACHE_USERS = "users";
 	String CACHE_FILE_ENTRIES = "file_entries";
+	String CACHE_MESSAGES = "messages";
```
Comment (junoyoon) [inline ngrinder-controller/src/main/java/org/ngrinder/common/constant/CacheConstants.java]: CACHE_USER_DEFINED_MESSAGES

File: ngrinder-controller/src/main/java/org/ngrinder/infra/config/UserMessageSource.java [hunk 3]
```diff
@@ -121,4 +122,12 @@ protected MessageFormat resolveCode(String code, Locale locale) {
 		MessageFormat resolved = langMessageMap.get(new LocaleAndCode(locale.getLanguage(), code));
 		return resolved == null ? langMessageMap.get(new LocaleAndCode("en", code)) : resolved;
 	}
+
+	public Map<String, String> getMessageMap(String locale) {
+		return langMessageMap
+			.entrySet()
+			.stream()
+			.filter(entry -> entry.getKey().locale.equals(locale))
+			.collect(Collectors.toMap(entry -> entry.getKey().code, entry -> entry.getValue().toPattern()));
```
Comment (junoyoon) [inline ngrinder-controller/src/main/java/org/ngrinder/infra/config/UserMessageSource.java]: It's not the dynamic. please create map it in @PostConstruct

**Comment by junoyoon** [comment]
LGTM

**Pull request merged**